src/merge_sort.py

```diff
@@ -0,0 +1,78 @@
+"""Merge Sort Module."""
+
+# MERGE SORT (MS)
```

Comment:
This style of commenting at the top should be contained in the upper triple quotes area:
"""Merge Sort Module.

blah blah blah...
"""